bucket/sorayomi.json

```
@@ -0,0 +1,39 @@
+{
+  "version": "0.2.3",
+  "description": "A flutter manga reader compatible with Tachiyomi extensions and backups",
+  "homepage": "https://github.com/Suwayomi/Tachidesk-Sorayomi",
+  "license": {
+    "identifier": "MPL-2.0",
+    "url": "https://github.com/Suwayomi/Tachidesk-Sorayomi/blob/main/LICENSE"
+  },
+  "notes": [
+    "We implore you to install",
+    "either Tachidesk-Server or Tachidesk-Server-preview",
+    "for Sorayomi to work"
+  ],
+  "suggest": {
+    "Tachidesk-Server": [
+      "extras/tachidesk-server",
+      "extras/tachidesk-server-preview"
+    ]
+  },
```

```suggestion
  "suggest": {
    "Tachidesk-Server": "extras/tachidesk-server",
  },
```

There is no `tachidesk-server-preview` in Extras bucket.
If you want to add it to *Scoop*, you can send `tachidesk-server-preview` to [Versions](https://github.com/ScoopInstaller/Versions) bucket.

---

updated changes